bin/__tests__/getIntegrationAccessToken.test.js

```diff
@@ -222,14 +224,14 @@ describe('getIntegrationAccessToken', () => {
       expect(mockAuth).toHaveBeenCalledWith(
         expectedAuthOptions({
           metaScopes: [
-            'https://scope.com/s/ent_reactor_extension_developer_sdk',
+            'https://scope.com/s/ent_reactor_sdk',
           ],
         })
       );
 
       expect(mockAuth).toHaveBeenCalledWith(
         expectedAuthOptions({
-          metaScopes: ['https://scope.com/s/ent_reactor_admin_sdk'],
+          metaScopes: ['https://scope.com/s/ent_reactor_admin'],
```

Comment:
We need to add an assertion for each of the metascopes (rather than just a couple).